From 7eae073fda17a94a36893337b5a5d639024d571c Mon Sep 17 00:00:00 2001
From: Petr Vorel <petr.vorel@gmail.com>
Date: Sun, 17 Nov 2019 12:53:22 +0100
Subject: [PATCH] nfsv4/acl: Remove unneeded malloc(sizeof(FILE))

FILE incomplete type under musl (comply C11 [1]).
malloc(sizeof(FILE)) is not needed anyway, thus remove.

This fixes build on musl (at least):
acl1.c:51:23: error: invalid application of 'sizeof' to incomplete type 'FILE' {aka 'struct _IO_FILE'}
  fptr = malloc(sizeof(FILE));

Fixes: 618fd23eb ("Initial add of acl testsuite from Bull")

[1] https://www.openwall.com/lists/musl/2019/01/16/1

Signed-off-by: Petr Vorel <petr.vorel@gmail.com>
[Upstream status: 7eae073fda17a94a36893337b5a5d639024d571c]
---
 testcases/network/nfsv4/acl/acl1.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/testcases/network/nfsv4/acl/acl1.c b/testcases/network/nfsv4/acl/acl1.c
index bdf0180da..aae9484e5 100644
--- a/testcases/network/nfsv4/acl/acl1.c
+++ b/testcases/network/nfsv4/acl/acl1.c
@@ -48,7 +48,6 @@ int do_file_op(char *filename)
 	result = 0;
 	FILE *fptr;
 	char str[256] = "./";
-	fptr = malloc(sizeof(FILE));
 
 	uid = geteuid();
 	strcat(str, filename);
-- 
2.23.0

